
Well, after fixing user-space apps here and there it became too annoying
and counter productive, since nowadays user-space expect full i386
style linux mapping anyways. So let's correct the alt and altgr mapping
in the kernel in the first place.

  - Rene Rebe - T2 project

--- drivers/macintosh/adbhid.c.orig	2004-08-31 16:50:10.000000000 +0200
+++ drivers/macintosh/adbhid.c	2004-08-31 18:09:07.000000000 +0200
@@ -78,7 +78,7 @@
 	 30, 31, 32, 33, 35, 34, 44, 45, 46, 47, 86, 48, 16, 17, 18, 19,
 	 21, 20,  2,  3,  4,  5,  7,  6, 13, 10,  8, 12,  9, 11, 27, 24,
 	 22, 26, 23, 25, 28, 38, 36, 40, 37, 39, 43, 51, 53, 49, 50, 52,
-	 15, 57, 41, 14, 96,  1, 29,125, 42, 58, 56,105,106,108,103,  0,
+	 15, 57, 41, 14, 96,  1, 29, 56, 42, 58,100,105,106,108,103,  0,
 	  0, 83,  0, 55,  0, 78,  0, 69,  0,  0,  0, 98, 96,  0, 74,  0,
 	  0,117, 82, 79, 80, 81, 75, 76, 77, 71,  0, 72, 73,124, 89,121,
 	 63, 64, 65, 61, 66, 67,123, 87,122, 99,  0, 70,  0, 68,101, 88,
